js/src/dropdown.js

```diff
@@ -1,3 +1,5 @@
+/* global Popper */
```

Comment:
Just wondering, why do you need to set it as global? Wouldn't it cleaner to import it with ESM and mark it as external dependency in your modules resolver?